src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MultipleCreateMutationTestBase.cs

```diff
@@ -0,0 +1,819 @@
+            string graphQLMutationName = "createbook";
+            string graphQLMutation = @"mutation {
+                                          createbook(
+                                            item: {
+                                              title: ""Book #1""
+                                              publishers: { name: ""Publisher #1"" }
+                                              reviews: [
+                                                {
+                                                  content: ""Book #1 - Review #1""
+                                                  website_users: { id: 5001, username: ""WebsiteUser #1"" }
+                                                }
+                                                { content: ""Book #1 - Review #2"", websiteuser_id: 1 }
+                                              ]
+                                              authors: [
+                                                { birthdate: ""2000-02-01"", name: ""Author #1"", royalty_percentage: 50.0 }
+                                                { birthdate: ""2000-01-02"", name: ""Author #2"", royalty_percentage: 50.0 }
+                                              ]
+                                            }
+                                          ) {
+                                            id
+                                            title
+                                            publishers {
+                                              id
+                                              name
+                                            }
+                                            reviews {
+                                              items {
+                                                book_id
+                                                id
+                                                content
+                                                website_users {
+                                                  id
+                                                  username
+                                                }
+                                              }
+                                            }
+                                            authors {
+                                              items {
+                                                id
+                                                name
+                                                birthdate
+                                              }
+                                            }
+                                          }
+                                        }";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            SqlTestHelper.PerformTestEqualJsonStrings(expectedResponse, actual.ToString());
+
+            // Validate that the records are created in the linking table
```

Comment:
which records? can you add a little more detail here?

Reply:
Updated